packages/flutter/lib/src/animation/curves.dart

```diff
@@ -1334,7 +1334,7 @@ class Curves {
   /// By default is the curve used to animate pages on iOS back to their original
   /// position if a swipe gesture is ended midway through a swipe.
   ///
-  /// {@animation 464 192 https://flutter.github.io/assets-for-api-docs/assets/animation/fast_linear_to_slow_ease_in.mp4}
+  /// {@animation 464 192 https://flutter.github.io/assets-for-api-docs/assets/animation/curve_fast_linear_to_slow_ease_out.mp4}
```

Comment:
It looks like this file is actually incorrectly named. While it is showing the right curve, it should actually be named "curve_fact_linear_to_slow_in.mp4". The curve is generated here: https://github.com/flutter/assets-for-api-docs/blame/master/packages/diagrams/lib/src/curve.dart#L475

Could you correct the naming there and then use the correct file name here?

Reply:
Yeah, I will commit it

Reply:
I have commited the correct name here and also created a pull request in the curve generator. (https://github.com/flutter/assets-for-api-docs/pull/111)